tests/acceptance/features/apiFullTextSearch/search.feature

```diff
@@ -83,3 +83,26 @@ Feature: full text search
       | old              |
       | new              |
       | spaces           |
+
+
+  Scenario Outline: search project space files by content
+    Given using spaces DAV path
+    And the administrator has assigned the role "Space Admin" to user "Alice" using the Graph API
+    And user "Alice" has created a space "project-space" with the default quota using the GraphApi
+    And user "Alice" has created a folder "spacesFolderWithFile/spacesSubFolder" in space "project-space"
+    And user "Alice" has uploaded a file inside space "project-space" with content "hello world from nepal" to "keywordAtStart.txt"
+    And user "Alice" has uploaded a file inside space "project-space" with content "saying hello to the world" to "spacesFolderWithFile/keywordAtMiddle.txt"
+    And user "Alice" has uploaded a file inside space "project-space" with content "nepal want to say hello" to "spacesFolderWithFile/spacesSubFolderkeywordAtLast.txt"
+    And user "Alice" has uploaded a file inside space "project-space" with content "namaste from nepal" to "hello.txt"
+    And using <dav-path-version> DAV path
+    When user "Alice" searches for "Content:hello" using the WebDAV API
+    Then the HTTP status code should be "207"
+    And the search result of user "Alice" should contain only these files:
+      | keywordAtStart.txt  |
+      | keywordAtMiddle.txt |
+      | keywordAtLast.txt   |
```

Comment:
How can this pass - `keywordAtLast.txt` does not exist, because it was accidentally called `spacesSubFolderkeywordAtLast.txt`

We need to check the code for this step and find anything that might be wrong with it. We don't want false positives (or false negatives)

Reply:
If remember correct, the code checks with the endswith method. Yeah, have to make it strict

Reply:
@saw-jan @nabim777 is someone working on this step refactoring in separate PR or in same PR?

Reply:
@amrita-shrestha I am doing the refactoring in separate PR

Reply:
See the separate PR that addresses the test code issue.
We can move forward and merge this PR for the new scenario.